openedx/features/survey_report/models.py

```diff
@@ -0,0 +1,29 @@
+"""
+Survey Report models.
+"""
+
+from django.db import models
+from jsonfield import JSONField
+
+
+class SurveyReport(models.Model):
+    """
+    This model stores information to automate the way of gathering impact data from the openedx project.
+
```

Comment:
It'll be helpful to have a description for each field here

Reply:
@mariajgrimaldi Thanks for your comments, I have already added it.